src/include/module/module/interface.h

```diff
@@ -0,0 +1,222 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
@softwarecki I'm guessing git not detect the move here and this is not a new file ?

Reply:
I moved some parts of `src/include/sof/audio/module_adapter/module/module_interface.h` to this file, so it's a copy rather than a move.

> Git does not do rename tracking nor copy tracking, which means **it doesn't record renames or copies**. What it does instead is **rename and copy detection**.
https://stackoverflow.com/a/1043566